.circleci/config.yml

```diff
@@ -98,6 +98,23 @@ jobs:
           paths:
             - release/centos_7_6
 
+  centos_8_3:
+    machine:
+      image: ubuntu-1604:202007-01
+    steps:
+      - attach_workspace:
+          at: /tmp/workspace
+      - run: docker build -f /tmp/workspace/.circleci/dockerfiles/centos-8.3 -t wake-centos-8.3 .
+      - run: |
+          cp /tmp/workspace/wake_*.tar.xz .
+          docker run --rm --mount type=bind,source=$PWD,target=/build --device /dev/fuse --cap-add SYS_ADMIN --security-opt apparmor:unconfined wake-centos-8.3 rpmbuild -ta --define "_rpmdir /build" wake_*.tar.xz
+          docker run --rm --mount type=bind,source=$PWD,target=/build wake-centos-8.3 /bin/sh -c "rpm -i x86_64/*.rpm; wake --no-workspace -x 'shellJob \"ls /\" Nil | getJobStdout'"
```

Comment:
Please change the `;` to `&&`. Otherwise an installation failure might still "pass" because wake is "installed" (half-way).